blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_3.py

```diff
@@ -0,0 +1,63 @@
+"""Formula 5.3 from NEN-EN 1993-5:2008 Chapter 5 - Ultimate limit state."""
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008 import NEN_EN_1993_5_2008
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS, KNM, MM3, MPA
+from blueprints.unit_conversion import N_TO_KN
+from blueprints.validations import raise_if_less_or_equal_to_zero
+
+
+class Form5Dot3DesignMomentResistanceClass3(Formula):
+    """Class representing a formula 5.3 for design moment resistance for Class 3 cross-sections."""
+
+    label = "5.3"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(
+        self,
+        beta_b: DIMENSIONLESS,
+        w_el: MM3,
+        f_y: MPA,
+        gamma_m_0: DIMENSIONLESS,
+    ) -> None:
+        """(Mc,Rd) Calculate design moment resistance of the cross-section (class 3) in [kNm/m]
+        based on NEN-EN 1993-5:2007(E) art. 5.2.2(2) formula 5.3.
```

Comment:
This is other NEN then mentioned in source_document, is this right? (Did not check it myself with the eurocode)